src/main/java/org/spongepowered/api/service/economy/EconomyService.java

```diff
@@ -69,49 +69,48 @@
     Set<Currency> getCurrencies();
 
     /**
-     * Gets the {@link UniqueAccount} with the specified {@link UUID}, if available.
+     * Returns whether a {@link UniqueAccount} exists with the specified {@link UUID}.
      *
-     * @param uuid The {@link UUID} of the account to get
-     * @return The {@link UniqueAccount}, if available.
+     * @param uuid The {@link UUID} of the account to check for
+     * @return Whether a {@link UniqueAccount} exists with the specified {@link UUID}
      */
-    Optional<UniqueAccount> getAccount(UUID uuid);
+    boolean hasAccount(UUID uuid);
 
     /**
-     * Gets the {@link Account} with the specified identifier, if available.
+     * Returns whether an {@link Account} with the specified identifier exists.
      *
-     * <p>The returned {@link Account} may be a {@link UniqueAccount} or
-     * a {@link VirtualAccount}, depending on the implementation.</p>
+     * <p>Depending on the implementation, the {@link Account} may be a {@link UniqueAccount} or
+     * a {@link VirtualAccount}.
      *
-     * @param identifier The identifier of the account to get
-     * @return The {@link VirtualAccount}, if available.
+     * @param identifier The identifier of the account to check for
+     * @return Whether an {@link Account} with the specified identifier exists
      */
-    Optional<Account> getAccount(String identifier);
-
+    boolean hasAccount(String identifier);
 
     /**
-     * Attempts to create a {@link UniqueAccount} for the user with the specified {@link UUID}.
+     * Gets the {@link UniqueAccount} for the user with the specified {@link UUID}.
      *
-     * <p>If an account already exists with the specified {@link UUID}, it will be
-     * returned. No further action will be taken.</p>
+     * <p>If an account does not already exists with the specified {@link UUID}, it will be
+     * created.</p>
      *
      * <p>Creation might fail if the provided {@link UUID} does not correspond to an actual
      * player, or for an implementation-defined reason.</p>
      *
-     * @param uuid The {@link UUID} of the account to create.
-     * @return The created {@link UniqueAccount}, if available.
+     * @param uuid The {@link UUID} of the account to get.
+     * @return The {@link UniqueAccount}, if available.
      */
-    Optional<UniqueAccount> createAccount(UUID uuid);
+    Optional<UniqueAccount> getAccount(UUID uuid);
```

Comment:
To play devil's advocate. You could say that all players always "have" an account even if it hasn't technically been created yet. However, to contradict myself that kind of discounts hasAccount method as that doesn't really make sense if you prescribe to the above notion. So in conclusion I agree with @Zidane unless hasAccount is removed.